tests/PayU/Alu/RequestTest.php

```diff
@@ -76,7 +76,9 @@ public function setUp()
 
         $card = new Card('5431210111111111', '11', 2016, 123, 'test');
 
-        $this->request = new Request($cfg, $this->order, $billing, $delivery, $user);
+        $fx = new FX('EUR', 0.2462);
+
+        $this->request = new Request($cfg, $this->order, $billing, $delivery, $user, $fx);
```

Comment:
Do we still have a test for when fx is not used and parameters should not be sent?

Reply:
Done!